Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: netdata
 home: https://github.com/netdata/netdata
-version: 1.2.0
-appVersion: v1.21.1
+version: 1.22.1
```

Comment:
Version is a semver of the helmchart, it wasnt equal to the netdata version before

Reply:
@prologic Looks like there's a bug in the update script.